### DISCUSSION TITLE
What is version 11.1.4 and why is it not listed?

### DISCUSSION THREAD

**Discussion by roxeteer** [vercel/next.js#34159] [Help]
Looks like version 11.1.4 was released to npm two weeks ago. It's not listed as a release here on GitHub. Is it something we should upgrade to in production or was it tagged in error?

**Comment by wwwdepot** [comment]
From commit history, it updated `node-fetch` to v2.6.7 https://github.com/vercel/next.js/commits/v11.1.4
